deploy/gcp/prod.tfvars

```diff
@@ -0,0 +1,5 @@
+tidb_operator_version = "v1.0.0-beta.3.start-immediate"
+tidb_operator_registry = "gcr.io/pingcap-tidb-alpha"
```

Comment:
How does this registry get updated?

Reply:
these 2 variables are an accidental commit from my testing.